src/redux/configurestore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import bookReduce from './books/books';
+import categoryReduce from './categories/categories';
+
+const rootReducer = combineReducers({
+  book: bookReduce,
+  categ: categoryReduce,
+});
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for this project, " `Combine both reducers into a root reducer by using configureStore function.` ", kindly use configureStore to combine your reducers.